### PR TITLE
Add SBOM handling action

### LINKED ISSUE #8: Add SBOM Handling Action
Add an action that download the full SBOM for the current project and puts it in the `$RELEASE_ASSETS` folder to be uploaded to the GitHub Release.  We'll need credentials from the AWS vault and the name of the project as a variable.

### PR DESCRIPTION
**Pull request by blink1073**
Fixes #8.

Tested with https://github.com/mongodb/winkerberos/pull/88

image

### CODE REVIEW

**Comment by blink1073** [comment]
TODO: we should update the SBOM first, to ensure it is up to date.

**Comment by blink1073** [comment]
> TODO: we should update the SBOM first, to ensure it is up to date.

We're going to defer this for now.

**Comment by alcaeus** [comment]
Heads up, I'll need a while until I can test this, as I'm only now migrating PHP to the v2 version of the tools. If another driver is in position to test this, I'd very much appreciate it. // cc @baileympearson

File: README.md [hunk 1]
```diff
@@ -107,6 +107,22 @@ It will create the file `$S3_ASSETS/authorized_publication.txt`
     token: ${{ github.token }}
 ```
 
+### Software Bill of Materials (SBOM)
+
+This action will download an Augmented SBOM file in `$RELEASE_ASSETS/sbom.json`.
+
+```yaml
+- name: Setup
+  uses: mongodb/drivers-github-tools/setup@v2
```
Comment (alcaeus) [inline README.md:116]:
```suggestion
  uses: mongodb-labs/drivers-github-tools/setup@v2
```

File: README.md [hunk 2]
```diff
@@ -107,6 +107,22 @@ It will create the file `$S3_ASSETS/authorized_publication.txt`
...
+  with:
+    ...
+
+- name: Create SBOM
+  uses: mongodb/drivers-github-tools/sbom@v2
```
Comment (alcaeus) [inline README.md:121]:
```suggestion
  uses: mongodb-labs/drivers-github-tools/sbom@v2
```

**Review by alcaeus — COMMENTED** [review]
Tested for PHPLIB, and the action itself works (but our Silk setup isn't complete yet, so there's no SBOM to be found). With that in mind, I think we can merge this and re-tag v2 to make it available.
Just left suggestions to fix the repository name in the readme ;)

**Pull request merged**